external-crates/move/crates/move-analyzer/src/analysis/parsing_analysis.rs

```diff
@@ -170,7 +170,8 @@ impl<'a> ParsingAnalysisContext<'a> {
                     if ignored_function(fun.name.value()) {
                         continue;
                     }
-                    earliest_member_loc = earliest_loc(earliest_member_loc, fun.loc);
+                    earliest_member_loc =
+                        earliest_loc(earliest_member_loc, fun.doc.loc().unwrap_or(fun.loc));
```

Comment:
This is a fix for "general" auto-import insertion that applies to auto-completion-style auto imports, but since it also applies to auto-fix style auto-imports, I figured I'd include it here